src/components/Photos.js

```diff
@@ -1,50 +1,46 @@
-import React, { Component } from 'react'
-import firebase from "../firebase"
-export default class Photos extends Component {
-  constructor(props){
-    super(props)
-    this.state={
-      photos:[]
-    }
-  }
-
-  showImage(){
-    let images=[]
-    firebase.firestore().collection(localStorage.getItem("user_id")).get().then(function(querySnapshot) {
-      querySnapshot.forEach(function(doc) {
-         
-
-          let imageData={
-            url:doc.data().url,
-            created:doc.data().added
-          }
-          images.push(imageData)
+import React, { useEffect, useState } from 'react';
+import firebase from '../firebase';
+export default function Photos() {
+  const [photos, setPhotos] = useState([]);
 
+  async function showImages() {
+    let images = [];
+     await firebase
+      .firestore()
+      .collection(localStorage.getItem('user_id'))
+      .get()
+      .then(function (querySnapshot) {
+        querySnapshot.forEach(function (doc) {
+          let imageData = {
```

Comment:
Are you redeclaring this `imageData` variable anywhere? If not just use `const` to declare it not `let`. 
Read through this short article https://css-tricks.com/let-vs-const/